src/components/executive-education-2u/components/ErrorPageContent.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Button, Col, Row } from '@edx/paragon';
+import { AppContext } from '@edx/frontend-platform/react';
+import { sendEnterpriseTrackEventWithDelay } from '@edx/frontend-enterprise-utils';
+import { ArrowBack } from '@edx/paragon/icons';
+
+import { ErrorPage } from '../../error-page';
+import ExecutiveEducation2UErrorIllustration from
+  '../../../assets/images/executive-education-2u/error-illustration.svg';
+import ContactAdminMailto from '../../contact-admin-mailto';
+
+export const showHelpfulLink = (failureCode) => {
+  const failureCodeShowLink = {
+    no_offer_available: true,
+    no_offer_with_enough_balance: true,
+    no_offer_with_enough_user_balance: true,
+    no_offer_with_remaining_applications: true,
+    system_error: false,
+    default: false,
+  };
+  return failureCodeShowLink[failureCode] ? failureCodeShowLink[failureCode] : failureCodeShowLink.default;
+};
+
+const ErrorPageContent = ({
+  className,
+  failureReason,
+  failureMessage,
+  httpReferrer,
+}) => {
+  const { enterpriseConfig: { uuid: enterpriseId } } = useContext(AppContext);
+
+  return (
+    <ErrorPage.Content className={classNames('text-center', className)}>
+      <Row>
+        <Col xs={12} lg={{ span: 10, offset: 1 }}>
+          <img
+            src={ExecutiveEducation2UErrorIllustration}
+            className="mb-4"
+            alt=""
+          />
+          <div className="executive-education-2u-error-heading mb-4">
+            <span className="executive-education-2u-error-heading-red mr-2">
+              We&apos;re sorry.
+            </span>
+            <span className="executive-education-2u-error-heading-black">
+              Something went wrong.
+            </span>
+          </div>
+          <p
+            className="executive-education-2u-error-heading-details mb-4"
+            data-testid="executive-education-2u-error-heading-details"
+          >
+            {failureMessage}
+          </p>
+          {httpReferrer && !showHelpfulLink(failureReason) && (
```

Comment:
nit: If we're branching on the result of `showHelpfulLink(failureReason)`, should probably call it once above and reuse the value.